src/custom.d.ts

```diff
@@ -1,13 +1,5 @@
 declare module '*.svg' {
   import React = require('react');
   export const ReactComponent: React.FC<React.SVGProps<SVGSVGElement>>;
-  const src: string;
-  export default src;
-}
-
-declare module '@sendbird/chat-ai-widget' {
-  import ChatAiWidget = require('@sendbird/chat-ai-widget');
-  const ChatAiWidget: ChatAiWidget.ChatAiWidget;
-  const Chat: ChatAiWidget.Chat;
-  export { ChatAiWidget, Chat };
+  export default ReactComponent;
```

Comment:
Could you explain this file change? I do not understand the removed nor added

Reply:
This changes from https://github.com/sendbird/chat-ai-widget/pull/180#discussion_r1582787203
It just type definition for svg import

```ts
// Before
import xxx from 'xxx.svg'; // string type

// After
import xxx from 'xxx.svg'; // ReactComponent type
```

Reply:
we dont need 
```
declare module '@sendbird/chat-ai-widget' {	
  import ChatAiWidget = require('@sendbird/chat-ai-widget');	
  const ChatAiWidget: ChatAiWidget.ChatAiWidget;	
  const Chat: ChatAiWidget.Chat;	
  export { ChatAiWidget, Chat };
```
?

Reply:
Yes, it seems to have been declared for use under `packages/*`, but it should not be used this way as it does not solve the fundamental problem.

As a fix for this, I have made modifications so that the package can correctly locate the type path.
https://github.com/sendbird/chat-ai-widget/pull/193/commits/85368205b69c46eba61222a9721ab39a428a6ca0